doc/nrf/protocols/matter/end_product/attestation.rst

```diff
@@ -169,16 +182,18 @@ Product Attestation Intermediate certificate
   It is used to create the DAC of the device and is provided with the factory data.
 
 Verifier
-  The Verifier itself is not a part of the DAC.
+  The Verifier itself is not a part of the DAC, although it is included in the factory data.
   See `Requirements for secured communication`_.
 
 Certification Declaration
-  A cryptographic document created by the CSA for each device type, used to confirm that a given type of device was certified.
+  This is |matter_cd_definition|.
   It contains a series of information required for Device Attestation, including Vendor ID, Certificate ID, certification type, optional information related to DAC, and other.
   CD is included in the attestation information packet send by the commissionee during the `Device Attestation procedure`_.
 
   .. note::
-      CD information is not located in the factory data, but in the application itself.
+      The CD information is not located in the factory data, but in the application itself.
+      This is because the CD information can change after the certification when you perform a device software update.
```

Comment:
The CD basically always changes if the software version changed and was re-certified. How about:
```suggestion
      This is because the CD changes after the new software version re-certification and it is applied to the device when you perform a device software update. 
```